precompiles/xtokens/src/tests.rs

```diff
@@ -876,7 +876,7 @@ fn transfer_multi_assets_is_not_sorted_error() {
 					PCall::transfer_multi_assets {
 						assets: assets.into(),
 						fee_item: 0,
-						destination: destination.clone(),
+						destination: destination.clone().into(),
```

Comment:
There are a lot of changes like this (adding a `.into()` call), do you know why?

Reply:
The reason is that I had to use new type pattern for the types like `MultiLocation` [here](https://github.com/moonbeam-foundation/moonbeam/pull/2427/files/6a9f43121577eb693528b34b3a8570fbe370bb15?diff=split&w=0#diff-32d3fd768b875dcf7bd291aafe9f213313f6c90f6322be43e6384e30e7f38b48R360).

Reply:
Exactly